browser/ui/webui/ai_chat/ai_chat_untrusted_conversation_ui.h

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_WEBUI_AI_CHAT_AI_CHAT_UNTRUSTED_CONVERSATION_UI_H_
+#define BRAVE_BROWSER_UI_WEBUI_AI_CHAT_AI_CHAT_UNTRUSTED_CONVERSATION_UI_H_
+
+#include <memory>
+
+#include "brave/components/ai_chat/core/common/mojom/ai_chat.mojom-forward.h"
+#include "brave/components/ai_chat/core/common/mojom/untrusted_frame.mojom-forward.h"
+#include "content/public/browser/webui_config.h"
+#include "ui/webui/mojo_web_ui_controller.h"
+
+class AIChatUntrustedConversationUIConfig : public content::WebUIConfig {
```

Comment:
please add comments explaining what this is used for

Reply:
Added:
```
// This Untrusted WebUI hosts the UI to display conversation entries, including
// ones generated by an LLM. It should not be granted more permissions than
// required to display the conversation entries. Anything requiring access
// to browser features should be done in the trusted UI.
```